gtests/test_nopython_callprofilerserializer.cpp

```diff
@@ -1,554 +0,0 @@
-#include <gtest/gtest.h>
```

Comment:
Do you need to port some of the C++ testing code to Python?

Reply:
Yes, I test the serialize result by python.